### PR TITLE
Bug 1924483: Bump Kubernetes to 1.21

### PR DESCRIPTION
**Pull request by jsafrane**
And `github.com/openshift/*` to master.

@openshift/storage

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,29 +4,26 @@ go 1.13
 
 require (
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
-	github.com/go-logr/logr v0.3.0 // indirect
```
Comment (bertinatto) [inline go.mod:7]: nit: since we are here, we could you update the `go` directive above to `go 1.16` (and re-run `go mod vendor`)

Comment (jsafrane) [reply go.mod:7]: Good catch, done

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**